src/main/resources/default/assets/tycho/libs/token-autocomplete/token-autocomplete.js.map

```diff
@@ -0,0 +1 @@
AM,CAAC,WAAW,CAAC,EAAE;wBACnC,MAAM,CAAC,WAAW,CAAC,OAAO,CAAC,UAAU,UAAsB;4BACvD,EAAE,CAAC,aAAa,CAAC,UAAU,CAAC,CAAC;wBACjC,CAAC,CAAC,CAAC;wBACH,IAAI,EAAE,CAAC,WAAW,CAAC,UAAU,CAAC,MAAM,IAAI,CAAC,IAAI,EAAE,CAAC,OAAO,CAAC,aAAa,EAAE;4BACnE,EAAE,CAAC,aAAa,CAAC;gCACb,EAAE,EAAE,IAAI;gCACR,KAAK,EAAE,YAAY;gCACnB,IAAI,EAAE,EAAE,CAAC,OAAO,CAAC,aAAa;gCAC9B,KAAK,EAAE,IAAI;gCACX,IAAI,EAAE,YAAY;gCAClB,WAAW,EAAE,IAAI;gCACjB,UAAU,EAAE,IAAI;gCAChB,qBAAqB,EAAE,IAAI;gCAC3B,eAAe,EAAE,IAAI;6BACxB,CAAC,CAAC;yBACN;qBACJ;gBACL,CAAC,CAAC;gBACF,IAAI,cAAc,GAAG,EAAE,CAAC,OAAO,CAAC,qBAAqB,CAAC,KAAK,CAAC,CAAC;gBAC7D,EAAE,CAAC,OAAO,CAAC,IAAI,CAAC,KAAK,EAAE,cAAc,EAAE,IAAI,CAAC,CAAC;gBAC7C,EAAE,CAAC,OAAO,CAAC,YAAY,GAAG,MAAM,CAAC;gBACjC,EAAE,CAAC,OAAO,CAAC,gBAAgB,CAAC,cAAc,EAAE,kBAAkB,CAAC,CAAC;gBAChE,EAAE,CAAC,OAAO,CAAC,IAAI,EAAE,CAAC;YACtB,CAAC;YAED;;;;eAIG;YACH,+BAAa,GAAb,UAAc,UAAsB;gBAChC,IAAI,OAAO,GAAG,IAAI,CAAC,QAAQ,CAAC,UAAU,CAAC,CAAC;gBAExC,IAAI,KAAK,GAAG,UAAU,CAAC,EAAE,IAAI,UAAU,CAAC,KAAK,CAAC;gBAC9C,IAAI,IAAI,GAAG,UAAU,CAAC,UAAU,IAAI,UAAU,CAAC,KAAK,IAAI,UAAU,CAAC,IAAI,CAAC;gBAExE,OAAO,CAAC,OAAO,CAAC,KAAK,GAAG,KAAK,CAAC;gBAC9B,OAAO,CAAC,OAAO,CAAC,IAAI,GAAG,IAAI,CAAC;gBAC5B,IAAI,UAAU,CAAC,IAAI,IAAI,IAAI,EAAE;oBACzB,OAAO,CAAC,OAAO,CAAC,IAAI,GAAG,UAAU,CAAC,IAAI,CAAC;iBAC1C;gBAED,IAAI,EAAE,GAAG,IAAI,CAAC;gBACd,OAAO,CAAC,gBAAgB,CAAC,OAAO,EAAE,UAAU,MAAa;oBACrD,IAAI,IAAI,IAAI,EAAE,CAAC,OAAO,CAAC,aAAa,EAAE;wBAClC,OAAO,IAAI,CAAC;qBACf;oBACD,IAAI,EAAE,CAAC,MAAM,CAAC,OAAO,CAAC,UAAU,IAAI,WAAW,CAAC,MAAM,EAAE;wBACpD,IAAI,OAAO,CAAC,SAAS,CAAC,QAAQ,CAAC,sCAAsC,CAAC,EAAE;4BACpE,EAAE,CAAC,MAAM,CAAC,MAAM,CAAC,KAAK,CAAC,KAAK,CAAC,CAAC;yBACjC;6BAAM;4BACH,EAAE,CAAC,MAAM,CAAC,MAAM,CAAC,QAAQ,CAAC,KAAK,EAAE,IAAI,EAAE,UAAU,CAAC,IAAI,EAAE,KAAK,CAAC,CAAC;yBAClE;qBACJ;yBAAM;wBACH,EAAE,CAAC,MAAM,CAAC,MAAM,CAAC,iBAAiB,EAAE,CAAC;wBACrC,IAAI,OAAO,CAAC,SAAS,CAAC,QAAQ,CAAC,sCAAsC,CAAC,EAAE;4BACpE,IAAI,WAAW,GAAG,EAAE,CAAC,MAAM,CAAC,MAAqB,CAAC;4BAClD,WAAW,CAAC,mBAAmB,CAAC,IAAI,CAAC,CAAC;yBACzC;6BAAM;4BACH,EAAE,CAAC,MAAM,CAAC,MAAM,CAAC,QAAQ,CAAC,KAAK,EAAE,IAAI,EAAE,UAAU,CAAC,IAAI,EAAE,KAAK,CAAC,CAAC;yBAClE;qBACJ;oBACD,EAAE,CAAC,gBAAgB,EAAE,CAAC;oBACtB,EAAE,CAAC,eAAe,EAAE,CAAC;gBACzB,CAAC,CAAC,CAAC;gBAEH,IAAI,IAAI,CAAC,SAAS,CAAC,aAAa,CAAC,uCAAuC,GAAG,IAAI,GAAG,IAAI,CAAC,KAAK,IAAI,EAAE;oBAC9F,OAAO,CAAC,SAAS,CAAC,GAAG,CAAC,sCAAsC,CAAC,CAAC;iBACjE;gBAED,IAAI,CAAC,WAAW,CAAC,WAAW,CAAC,OAAO,CAAC,CAAC;gBACtC,IAAI,CAAC,eAAe,EAAE,CAAC;gBAEvB,EAAE,CAAC,MAAM,CAAC,GAAG,CAAC,kBAAkB,EAAE,UAAU,CAAC,CAAC;YAClD,CAAC;YAeL,cAAC;QAAD,CAAC,AArRqB;QAwQX,kBAAe,GAAuB,UAAU,UAAsB;YACzE,IAAI,MAAM,GAAG,QAAQ,CAAC,aAAa,CAAC,IAAI,CAAC,CAAC;YAC1C,MAAM,CAAC,WAAW,GAAG,UAAU,CAAC,eAAe,IAAI,UAAU,CAAC,KAAK,IAAI,UAAU,CAAC,IAAI,CAAC;YAEvF,IAAI,UAAU,CAAC,WAAW,EAAE;gBACxB,IAAI,WAAW,GAAG,QAAQ,CAAC,aAAa,CAAC,OAAO,CAAC,CAAC;gBAClD,WAAW,CAAC,WAAW,GAAG,UAAU,CAAC,qBAAqB,IAAI,UAAU,CAAC,WAAW,CAAC;gBACrF,WAAW,CAAC,SAAS,CAAC,GAAG,CAAC,2CAA2C,CAAC,CAAC;gBACvE,MAAM,CAAC,WAAW,CAAC,WAAW,CAAC,CAAC;aACnC;YAED,OAAO,MAAM,CAAC;QAClB,CAAC;YACJ;IACL,wBAAC;CAAA,AAl0BD,IAk0BC"}
```

Comment:
Optional: This file is not needed when the .ts file is not also included. Because of this i would omit it.

Reply:
or should we include the .ts file for easier debugging?

Reply:
Can you test if this works?